src/components/events-table.js

```diff
@@ -61,24 +61,46 @@ function inputDataRow(processEvent) {
 }
 
 const EventsTable = ({ taskData }) => {
-    const [eventPredefinedFilter, setEventPredefinedFilter] = useState([]);
+    const [
+        eventLevelPredefinedFilter,
+        setEventLevelPredefinedFilter,
+    ] = useState([]);
+    const [eventLogPredefinedFilter, setEventLogPredefinedFilter] = useState(
+        []
+    );
     const [levelFilter, setLevelFilter] = useState([]);
     const [logFilter, setLogFilter] = useState([]);
 
+    const loadPredefinedFilter = (
+        predefinedFilter,
+        setPredefinedFilter,
+        setActiveFilter
+    ) => {
+        setPredefinedFilter(predefinedFilter);
+        let activePredefinedFilter = [];
+        predefinedFilter.forEach((f) => {
+            if (f.defaultChecked && f.defaultChecked === true) {
+                activePredefinedFilter.push(...f.filterValue);
+            }
+        });
+        activePredefinedFilter = Array.from(new Set(activePredefinedFilter));
+        setActiveFilter(activePredefinedFilter);
+    };
+
     useEffect(() => {
         fetch('process-metadata.json')
             .then((res) => res.json())
             .then((res) => {
-                const filter = res.eventPredefinedFilter;
-                setEventPredefinedFilter(filter);
-                let newtoFilter = [];
-                filter.forEach((f) => {
-                    if (f.defaultChecked && f.defaultChecked === true) {
-                        newtoFilter.push(...f.filterValue);
-                    }
-                });
-                newtoFilter = Array.from(new Set(newtoFilter));
-                setLogFilter(newtoFilter);
+                loadPredefinedFilter(
```

Comment:
This call does nothing at the moment because all your level filters are defaultChecked : false.
Are you sure that one day we will want one of these filters to be checked by default?
if yes then I understand the anticipated treatment otherwise I might as well avoid unnecessary treatment

Reply:
In SWE, the "Summary" predefined filter is checked by default.

Reply:
I was talking about the call for eventLevelPredefinedFilter and not the eventLogPredefinedFilter